### PR TITLE
Removed a repeated cell and added little enhancement

### PR DESCRIPTION
**Pull request by nvinayvarma189**

### CODE REVIEW

File: .gitmodules [hunk 1]
```diff
@@ -1,3 +1,3 @@
 [submodule "aima-data"]
 	path = aima-data
-	url = https://github.com/aimacode/aima-data.git
+	url = https://github.com/aimacode/aima-data
```
Comment (ad71) [inline .gitmodules]: Even though `.git` is not required, I think it should not be removed as it indicates that the directory is a _bare_ git repository

Comment (MrDupin) [reply .gitmodules]: I don't think it matters that much, any way works for me.

Comment (nvinayvarma189) [reply .gitmodules]: I once unknowingly removed the `.git` and it's been following me in every PR. I'm sorry for that @ad71  @MrDupin . Please ignore that and pull only the changes from search notebook.Thanks.

Comment (ad71) [reply .gitmodules]: Unfortunately, pull requests don't work that way. You can just push a new commit to this branch, but as this is a trivial issue, you can just let it be. 
I personally find [this](https://stackoverflow.com/questions/29049650/github-fork-your-branch-is-5-commits-ahead-how-to-clean-this-without-pushing) very useful for merge issues. [`git cherry-pick`](https://git-scm.com/docs/git-cherry-pick) can also be used. Thanks!

**Comment by norvig** [comment]
This PR has conflicts.

**Comment by nvinayvarma189** [comment]
closing this one and I opened a fresh PR

**Pull request closed without merging**